src/Components/ExternalResult/FacilitiesSelectDialogue.tsx

```diff
@@ -30,7 +30,7 @@ const FacilitiesSelectDialog = (props: Props) => {
         showAll={false}
         multiple={false}
       />
-      <div className="mt-4 flex justify-between">
+      <div className="mt-4 flex flex-col">
```

Comment:
@AbhishekPSingh07 in the issue it was mentioned it should be in new line only in mobile view. Also, I've added some spacing between the two buttons.

```suggestion
      <div className="mt-4 flex flex-col sm:flex-row gap-2">
```